test/libsolidity/syntaxTests/conversion/explicit_conversion_from_event.sol

```diff
@@ -0,0 +1,34 @@
+interface MyInterface {
+    enum MyEnum { E1, E2 }
+}
+
+contract Test {
+    function testFunction(string memory) external returns (uint) {}
+
+    event CustomEvent1(
+        uint256,
+        bool, 
+        bool[],
+        address payable,
+        MyInterface,
+        MyInterface.MyEnum,
+        function (string memory) external returns (uint)
+    );    
```

Comment:
`chk_coding_style` is failing because of this. Please configure your editor to strip trailing whitespace.
```suggestion
    );
```